media-types.md

```diff
@@ -62,6 +62,6 @@ The following figure shows how the above media types reference each other:
 ![](img/media-types.png)
 
 [Descriptors](descriptor.md) are used for all references.
-The manifest list being a "fat manifest" references one or more image manifests per target platform. An image manifest references exactly one target configuration and possibly many layers.
+The image-index being a "fat manifest" references one or more image manifests per target platform. An image manifest references exactly one target configuration and possibly many layers.
```

Comment:
You don't hyphenate “image index” in most of the other places where you mention it.  I don't care if you hyphenate it or not, but it's probably worth picking one way and staying consistent.